docs/StardustDocs/topics/info/tail.md

```diff
@@ -0,0 +1,55 @@
+# tail
+
+
+<web-summary>
+Discover `tail` operation in Kotlin Dataframe.
+</web-summary>
+
+<card-summary>
+Discover `tail` operation in Kotlin Dataframe.
+</card-summary>
+
+<link-summary>
+Discover `tail` operation in Kotlin Dataframe.
+</link-summary>
+
+<!---IMPORT org.jetbrains.kotlinx.dataframe.samples.api.info.TailSamples-->
+
+Returns a [`DataFrame`](DataFrame.md) with the last `numRows` rows.
```

Comment:
I'd write this like "with the final 5 rows. You can also specify another number of rows." Something like that. It promotes the idea of using `tail()` instead of `takeLast(5)` and why you might want to use it instead. (So to get a quick view of the end of the dataframe)